src/khoj/routers/api_chat.py

```diff
@@ -250,6 +251,9 @@ async def chat(
 
     if conversation_commands == [ConversationCommand.Default]:
         conversation_commands = await aget_relevant_information_sources(q, meta_log)
+        mode = await aget_relevant_modes(q, meta_log)
```

Comment:
We should (at some point?) parallelize calls to `aget_relevant_output_modes` and `aget_relevant_information_sources` to reduce response latency

Reply:
Yes, I'm tracking a ticket for this separately.